src/sentry/static/sentry/app/views/settings/components/settingsProjectItem.jsx

```diff
@@ -0,0 +1,99 @@
+import styled from 'react-emotion';
+
+import React from 'react';
+
+import createReactClass from 'create-react-class';
+
+import {update} from '../../../actionCreators/projects';
+import ApiMixin from '../../../mixins/apiMixin';
+import TooltipMixin from '../../../mixins/tooltip';
+import Link from '../../../components/link';
+import ProjectLabel from '../../../components/projectLabel';
+import SentryTypes from '../../../proptypes';
+
+const StyledH5 = styled('h5')`
+  margin: 0;
+  a {
```

Comment:
Instead of nesting selectors, I think using a sep styled component makes more sense here since the style of `a` shouldn't need to depend on it being a child of an `h5`.